db/tunnel.go

```diff
@@ -198,6 +207,14 @@ func handleConnToTunnel(sshClient *ssh.Client, dbUrl *url.URL, sock net.Conn, er
 	fmt.Printf("End of connection [%d]\n", connID)
 	// Connection timeout
 	if err != nil && strings.Contains(err.Error(), "use of closed network") {
+
+		// If the connection has been closed by the CLIENT, we must stop here and return a nil error
+		// If the connection has been closed by the SERVER, we must return a errTimeout and retry the connection
```

Comment:
Isn't it legitimate that the server closes the connection sometimes?

Reply:
Not on this connection, if the server closed the conneciton, it will break the conneciton at line 191.  Here it's the ssh conneciton that broke.